requirements/base.txt

```diff
@@ -1,3 +1,3 @@
-Django>=2.2,<=4.0
+Django>=3.2,<=4.1
```

Comment:
I'm not entirely sure there are plans to change the renderer anytime soon. So it might be fine to just specify the lower bound and ignore the upper bound. Perhaps we can add some tests with the "latest stable" of django as a pre-emptive warning if something does break